addons/point_of_sale/tests/test_anglo_saxon.py

```diff
@@ -155,8 +158,30 @@ def test_2041208(self):
             'amount_tax': 0,
             'amount_paid': 0,
             'amount_return': 0,
+        }
+
+        self.pos_order_pos0 = self.PosOrder.create(pos_order_values)
+
+    def test_2041208_without_invoice(self):
```

Comment:
rename test_fifo_valuation_no_invoice and the other one test_fifo_valuation_with_invoice

Reply:
add a docstirng
"""Register a payment and validate a session after selling a fifo product"""

Reply:
dosctring not added